src/models/interfaces.ts

```diff
@@ -1,4 +1,5 @@
 export interface IMovie {
     text: string,
     rating: number,
+    description?: string,
```

Comment:
Nit: this is an optional prop, but in the component, you're applying default logic to ensure it is always set to at least an empty string. In this case, you might as well make it a required prop, or modify the default logic in the component to set `undefined` if the user has not provided a description.

Reply:
Updated description prop to be required. Thanks for update.